### PR TITLE
Disables script and adds a chat message telling players to install CreateTweaker if it isn't installed but Create is

### PR DESCRIPTION
**Pull request by MobiusFlip**

### CODE REVIEW

File: scripts/create_compat.zs [hunk 1]
```diff
@@ -1,6 +1,15 @@
 // Create integration provided by MobiusFlip
-
 #snip modnotloaded create
+
+#snip modloaded createtweaker
+import crafttweaker.api.events.CTEventManager;
+CTEventManager.register<crafttweaker.api.event.entity.player.MCPlayerLoggedInEvent>((event) => {
+     var player = event.player;
+     player.sendMessage("You need create tweaker dummy: https://www.curseforge.com/minecraft/mc-mods/createtweaker");
```
Comment (Vazkii) [inline scripts/create_compat.zs:8]: Less aggressive please lmao

**Review by Vazkii — CHANGES_REQUESTED** [review]

File: scripts/create_compat.zs (hunk 1 above)
Comment (Vazkii) [reply scripts/create_compat.zs:8]: While hilarious, the message doesn't actually explain *why* you need CreateTweaker. Sure, you can derive it from context, but maybe someone added create previously and forgot, or a friend handed them the pack. More information is better.

Comment (MobiusFlip) [reply scripts/create_compat.zs:8]: Pfft yeah fair. Changing it now

Comment (MobiusFlip) [reply scripts/create_compat.zs]: Ok, done

**Pull request merged**